custom/enikshay/integrations/nikshay/field_mappings.py

```diff
@@ -21,13 +21,15 @@
 }
 
 treatment_support_designation = {
-    'health_worker': '1',
-    'tbhv': '2',
-    'asha_or_other_phi_hw': '3',
-    'aww': '4',
-    'ngo_volunteer': '5',
-    'private_medical_pracitioner': '6',
-    'other_community_volunteer': '7',
+    'health_worker': '1',  # Health Worker
+    'tbhv': '2',  # TBHV
+    'asha': '3',  # ASHA
+    'asha_or_other_phi_hw': '3',  # ASHA
+    'aww': '4',  # AWW
+    'ngo_volunteer': '5',  # NGO VOLUNTEER
+    'private_medical_pracitioner': '6',  # PRIVATE PRACTICENER
```

Comment:
nit: sp in the comment: `practitioner` (and the key, but I'm assuming that's intentional)

Reply:
didnt realise.. copied it from the [form](https://enikshay.in/a/enikshay/apps/view/3c005532d11e463680b3f9124ea5b5a8/form/bf2473304010ef7518ad99b24eb6eafc3bcfab6d/source/) and its actually `private_medical_pracitioner` only.